crates/runtime/examples/rps.rs

```diff
@@ -96,15 +96,15 @@ fn main() -> EyreResult<()> {
         [0; 32],
     );
     let create_keypair_outcome = run(&file, "create_keypair", cx, &mut storage, &limits)?;
-    dbg!(&create_keypair_outcome);
+    &create_keypair_outcome;
```

Comment:
is this intended? 🤔

Reply:
great clippy or whatever was introduced

Reply:
will revert all the changes not related to my code

Reply:
reverted